### PR TITLE
GLM: remove last Math::Vec3 in gamelogic

### PR DESCRIPTION
**Pull request by slipher**
There were 2 files left using it. For one of them I picked a commit from @bmorel and the other I wrote myself since on the glm-complete branch there were a bunch of commits and whitespace changes mixed in.

### CODE REVIEW

**Review by DolceTriade — APPROVED** [review]

**Comment by DolceTriade** [comment]
needs to fix build failure tho

File: src/sgame/components/TurretComponent.cpp [hunk 1]
```diff
@@ -125,31 +127,31 @@ bool TurretComponent::MoveHeadToTarget(int timeDelta) {
...
 			"Aiming. Elapsed: %d ms. Delta: %.2f. Max: %.2f. Old: %s. New: %s. Reached: %s.",
 			timeDelta, deltaAngles, maxAngleChange, oldRelativeAimAngles, relativeAimAngles, targetReached
 		);
 	}
 
 	// TODO: Move gentity_t.buildableAim to BuildableComponent.
-	Vec3 absoluteAimAngles = RelativeAnglesToAbsoluteAngles(relativeAimAngles);
-	absoluteAimAngles.Store(entity.oldEnt->buildableAim);
+	glm::vec3 absoluteAimAngles = RelativeAnglesToAbsoluteAngles(relativeAimAngles);
+	absoluteAimAngles = VEC2GLM(entity.oldEnt->buildableAim);
```
Comment (necessarily-equal) [inline src/sgame/components/TurretComponent.cpp:139]: This isn't equivalent. You want 
```suggestion
	VectorCopy(absoluteAimAngles, entity.oldEnt->buildableAim);
```

File: src/sgame/components/TurretComponent.cpp [hunk 2]
```diff
@@ -275,20 +277,20 @@ Vec3 TurretComponent::RelativeAnglesToAbsoluteAngles(const Vec3 relativeAngles)
 	QuatToAngles(absoluteRotation, absoluteAngles);
 
 	/*turretLogger.Debug("RelativeAnglesToAbsoluteAngles: %s → %s. Torso angles: %s.",
-		Utility::Print(relativeAngles), Utility::Print(Vec3::Load(absoluteAngles)), TorsoAngles()
+		Utility::Print(relativeAngles), Utility::Print(glm::vec3::Load(absoluteAngles)), TorsoAngles()
```
Comment (necessarily-equal) [inline src/sgame/components/TurretComponent.cpp:280]: This isn't used. I guess you intended this:
```suggestion
		Utility::Print(relativeAngles), Utility::Print(VEC2GLM(absoluteAngles)), TorsoAngles()
```
but this code is commented and there is probably no `Utility::Print(glm::vec3);`

**Review by necessarily-equal — CHANGES_REQUESTED** [review]

File: src/sgame/components/TurretComponent.cpp (hunk 2 above)
Comment (slipher) [reply src/sgame/components/TurretComponent.cpp]: Deleted the commented-out code.

**Pull request merged**